### PR TITLE
Rishitha_bura_authorprofilepage

### PR DESCRIPTION
**Pull request by rishithab48**
- Created Published Books front-end
- Authors can add their published books to their database and display it on their profile.
- Author Verification will save the Author's specified name.

### CODE REVIEW

File: src/components/PublishedBooks/AddNewPublishedBook.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        </div>
+        <div>Book Title:
+        <input
+                  type="text"
+                  id="book"
+                  placeholder="Enter the book name"
+                  className="text-sm p-2 border-2 border-gray-300 rounded-md"
+                />
+        </div>
+        <div>Author name:
```
Comment (racecn) [inline src/components/PublishedBooks/AddNewPublishedBook.js:18]: Text shouldnt be a direct unwrapped child of a div if we can avoid it. can we use an h3 or something?

File: src/components/PublishedBooks/AddPublishedBooksModal.js [hunk 2]
```diff
@@ -1,45 +1,125 @@
...
+      
+  } else {
+      console.log("AuthorName does not exist in AuthorsFromBook");
+  }
+  // onClose();
+};
+  return (
+    <>
+      <div className="flex items-center justify-center fixed top-0 left-0 right-0 bottom-0  bg-gradient-to-b from-maroon z-10">
+        <div className="overlay-inner rounded-lg bg-lightcolor w-700 h-550 p-6 relative box-border ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600">
```
Comment (racecn) [inline src/components/PublishedBooks/AddPublishedBooksModal.js]: tailwind width and height only go up to w-96 and h-96 unless you've defined them in the tailwind.config as 

`module.exports = {
  theme: {
    extend: {
      width: {
        '500': '500px',
      },
    },
  },`

**Review by racecn — CHANGES_REQUESTED** [review]

File: src/components/PublishedBooks/AddNewPublishedBook.js (hunk 1 above)
Comment (rishithab48) [reply src/components/PublishedBooks/AddNewPublishedBook.js:18]: Yeah, I still need to work on that page. I just put it there as a base for me to work on it later.

File: src/components/PublishedBooks/AddPublishedBooksModal.js (hunk 2 above)
Comment (rishithab48) [reply src/components/PublishedBooks/AddPublishedBooksModal.js]: I am actually not sure why that happened. I copied from an already existing code in our project. But I'll fix it.

Comment (rishithab48) [reply src/components/PublishedBooks/AddPublishedBooksModal.js]: I removed those lines. The sizing is coming from a CSS file.

**Review by racecn — APPROVED** [review]

**Pull request merged**